00-RELEASENOTES

```diff
@@ -1,16 +1,165 @@
+* Manual failover vote is not limited by two times the node timeout (#1305)
+* Automatic failover vote is not limited by two times the node timeout (#1356)
+
+Behavior Changes
+================
+* Rax size tracking (#688)
+* Take hz into account in activerehashing to avoid CPU spikes (#977)
+* Incr expired_keys if the expiration time is already expired (#1517)
+* Fix replica not able to initiate election in time when epoch fails (#1009)
+* Revert "Decline unsubscribe related command in non-subscribed mode" (#1265)
+* Make FUNCTION RESTORE FLUSH flush async based on lazyfree-lazy-user-flush (#1254)
+* Allow `MEMORY MALLOC-STATS` and `MEMORY PURGE` during loading phase (#1317)
+
+Logging and Tooling Improvements
+================================
+* Configurable log and timestamp formats (logfmt, ISO8601) (#1022)
+* Remove the restriction that cli --cluster create requires at least 3 primary nodes (#1075)
+* Add CMake build system for valkey (#1196)
+* Add short client info log to CLUSTER MEET / FORGET / RESET commands (#1249)
+* Support for reading from replicas in valkey-benchmark (#1392)
+* valkey-cli: re-select previously selected database after reconnect (#1694)
+* valkey-cli auto-exit from subscribed mode (#1432)
+
+Bug Fixes
+=========
+* Fix module / script call CLUSTER SLOTS / SHARDS fake client check crash (#1063)
+* Use _Thread_local to solve threads.h build issue (#1053)
+* Fix RDMA build dependency (#1074)
+* Fix primary crash when processing dirty slots during shutdown wait / failover wait / client pause (#1131)
+* Fix empty response for ACL CAT category subcommand for module defined categories (#1140)
+* Reset repl_down_since to zero only on state change (#1149)
+* Add 'WithDictIndex' expiry API and update RANDOMKEY command (#1155)
+* Fix FUNCTION KILL error message being displayed as SCRIPT KILL (#1171)
+* Add scoped RDB loading context and immediate abort flag (#1173)
+* Fix SORT GET to ignore special pattern # in cluster slot check (#1182)
+* Mark the node as FAIL when the node is marked as NOADDR and broadcast the FAIL (#1191)
+* [Bug Fix] Optimize RDB Load Performance and Fix Cluster Mode Resizing (#1199)
+* RDMA: Delete keepalive timer on closing (#1237)
+* Fix incorrect cache_memory reset in functionsLibCtxClear (#1255)
+* Log as primary role (M) instead of child process (C) during startup (#1282)
+* Fix empty primary may have dirty slots data due to bad migration (#1285)
+* Fix LRU crash when getting too many random lua scripts (#1310)
+* RDMA: Fix dead loop when transfer large data (20KB) (#1386)
+* Fix: restore replica output bytes stat update (#1486)
+* Use the correct command proc for the LOOKUP_NOTOUCH exception in lookupKey (#1499)
+* Fix for when active rehashing kvstore dictionaries are replaced by defrag (#1512)
+* Fix module LatencyAddSample still work when latency-monitor-threshold is 0 (#1541)
+* Fix default value of primary-reboot-down-after-period in sentinel.conf (#1040)
```

Comment:
```suggestion
* Mark the node as FAIL when the node is marked as NOADDR and broadcast the FAIL (#1191)
* [Bug Fix] Optimize RDB Load Performance and Fix Cluster Mode Resizing (#1199)
* Log as primary role (M) instead of child process (C) during startup (#1282)
* Fix empty primary may have dirty slots data due to bad migration (#1285)
* RDMA: Fix dead loop when transfer large data (20KB) (#1386)
```

Removing all the ones that are backported to 8.0, so they aren't bug fixes for this version technically.

Reply:
This wasn't fully addressed, some of these were left, was there a reason for this?